plonky2x/src/frontend/eth/storage/vars/storage.rs

```diff
@@ -192,10 +190,9 @@ impl CircuitVariable for EthLogVariable {
     }
 
     fn from_variables(variables: &[Variable]) -> Self {
-        let mut var_buffer = VariableBuffer::new(variables);
```

Comment:
`VariableBuffer` doesn't seem to be supported by `circuit.test_default_serializers`